src/Service/Parser/FunctionParser/ExplicitFunctionDefinitionInValueParser.php

```diff
@@ -10,15 +10,15 @@ class ExplicitFunctionDefinitionInValueParser extends AbstractFunctionParser
     /**
      * @param string|array $rawValue Raw value from query parameters
      */
-    public function supportsParameters(array $queryParameters, string $rawColumn, $rawValue): bool
+    protected function supportsParameters(array $queryParameters, string $rawColumn, $rawValue): bool
```

Comment:
Sidenote - restricting method visibility this generally means breaking change and release of new major version :-o. But I don't insist on enforcing it here.

(Do we have the BC break check job installed here?)

Reply:
Well for first, it is not tagged yet with these changes - so no BC would be needed.
And for second.. I see these components as `internal` - it is not possible to implement it easily, but if it would be possible, I would mark whole class as `private` so as I see it, no one should use these in their code directly.

And we dont have any BC break check (_yet?_) - just my head 😉

Reply:
Oh, I didn't noted the `@internal`. Then I guess this would be OK even with public API changed.

This may also be supported by BC break check in the future: https://github.com/Roave/BackwardCompatibilityCheck/issues/51